### PR TITLE
docs: cache tweets, version bumps, layout fix

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
Bumped a few versions too while I'm at it

**Comment by juliusmarminge** [comment]
Reduced headers in the Folder Structure to fix the line-breaks on small devices, i think they're plenty big still

File: cli/package.json [hunk 1]
```diff
@@ -58,6 +58,6 @@
     "prettier-plugin-tailwindcss": "^0.1.13",
     "tsup": "^6.2.3",
     "type-fest": "^3.0.0",
-    "typescript": "^4.8.4"
+    "typescript": "^4.9.3"
```
Comment (c-ehrlich) [inline cli/package.json:61]: 🔥

File: www/src/pages/en/folder-structure.md [hunk 2]
```diff
@@ -61,170 +61,170 @@ Further down, the description of each folder indicates its purpose and if it is
...
 
-### `src/pages/api`
+#### `src/pages/api`
 
 The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file (with Prisma) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file (with Next-Auth) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
 
 <sub>(With NextAuth.js, tRPC or tRPC + Prisma)</sub>
 
-### `src/pages/api/auth/[...nextauth].ts`
+#### `src/pages/api/auth/[...nextauth].ts`
```
Comment (c-ehrlich) [inline www/src/pages/en/folder-structure.md:88]: agreed the size is good on these

**Review by c-ehrlich — APPROVED** [review]

**Pull request merged**